eng/pipelines/docindex.yml

```
@@ -10,6 +10,10 @@ jobs:
       DocRepoLocation: $(Pipeline.Workspace)/docs
       DocRepoOwner: MicrosoftDocs
       DocRepoName: azure-docs-sdk-node
+      containerRegistry: azuresdkimages.azurecr.io
+      imageName: jsrefautocr
+      tag: latest
```

```suggestion
      docValidationImageId: azuresdkimages.azurecr.io/jsrefautocr:latest
```

Then just pass that to the other places.

---

Depending on what exactly the image is doing it might also be docBuildImageId.